### PR TITLE
feat(config): add Yale Assure 2 Biometric Deadbolt locks

### PR DESCRIPTION
**Pull request by oandrew**
Add configs for Yale Assure 2 Biometric Deadbolt locks:
```
YRD410-F-ZW3 Keyed Push Button Biometric Deadbolt
YRD420-F-ZW3 Keyed Touch Screen Biometric Deadbolt
YRD430-F-ZW3 Keyless Push Button Biometric Deadbolt
YRD450-F-ZW3 Keyless Touch Screen Biometric Deadbolt
```

Product IDs / Configuration IDs based on the official manual: https://products.z-wavealliance.org/ProductManual/File?folder=&filename=cert_portal/certs/859/YRD420_F_ZW3_MNL_1.pdf

Use a single device config file since they are functionally identical and differ only by keyed/keyless and push button/touch screen options.

Tested with my own `YRD420-F-ZW3`

Closes: #6611
Adds most of the locks from #6503

### CODE REVIEW

**Comment by oandrew** [comment]
@AlCalzone  PTAL when you have a chance

File: packages/config/config/devices/0x0129/yrd4x0-f-zw3.json [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"$import": "templates/yale_template.json#volume_two_options"
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0129/yrd4x0-f-zw3.json]: According to the manual, this has 3 options

File: packages/config/config/devices/0x0129/yrd4x0-f-zw3.json [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+			"#": "2",
+			"$import": "templates/yale_template.json#auto_relock"
+		},
+		{
+			"#": "3",
+			"$import": "templates/yale_template.json#auto_relock_time_180"
+		},
+		{
+			"#": "4",
+			"$import": "templates/yale_template.json#wrong_code_limit_3_to_10"
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0129/yrd4x0-f-zw3.json]: According to the manual, the default value is 5

File: packages/config/config/devices/0x0129/yrd4x0-f-zw3.json [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+		},
+		{
+			"#": "11",
+			"$import": "templates/yale_template.json#one_touch",
+			"defaultValue": 255
+		},
+		{
+			"#": "12",
+			"$import": "templates/yale_template.json#privacy_button"
+		},
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0129/yrd4x0-f-zw3.json]: This one is not documented in the manual

File: packages/config/config/devices/0x0129/yrd4x0-f-zw3.json [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+		{
+			"#": "13",
+			"$import": "templates/yale_template.json#lock_status_led"
+		},
+		{
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0129/yrd4x0-f-zw3.json:90]: Param 18 is missing here, but documented.

File: packages/config/config/devices/0x0129/yrd4x0-f-zw3.json [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+			"#": "19",
+			"$import": "templates/yale_template.json#dps_alarm"
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0129/yrd4x0-f-zw3.json]: Default is enabled

**Review by AlCalzone — CHANGES_REQUESTED** [review]
I'm not entirely sure if we should include the actual device labels (this can be done conditionally), split the file into 4, or keep it as is. Other than that:

File: packages/config/config/devices/0x0129/yrd4x0-f-zw3.json (hunk 2 above)
Comment (oandrew) [reply packages/config/config/devices/0x0129/yrd4x0-f-zw3.json]: Current default was 5 but according to the manual the default is 3. Fixed

File: packages/config/config/devices/0x0129/yrd4x0-f-zw3.json (hunk 3 above)
Comment (oandrew) [reply packages/config/config/devices/0x0129/yrd4x0-f-zw3.json]: Removed

File: packages/config/config/devices/0x0129/yrd4x0-f-zw3.json (hunk 4 above)
Comment (oandrew) [reply packages/config/config/devices/0x0129/yrd4x0-f-zw3.json:90]: Added it + fixed incorrect `minValue` in the `yale_template.json`

File: packages/config/config/devices/0x0129/yrd4x0-f-zw3.json (hunk 5 above)
Comment (oandrew) [reply packages/config/config/devices/0x0129/yrd4x0-f-zw3.json]: Fixed

File: packages/config/config/devices/0x0129/yrd4x0-f-zw3.json (hunk 1 above)
Comment (oandrew) [reply packages/config/config/devices/0x0129/yrd4x0-f-zw3.json]: Fixed + added correct `defaultValue`

**Comment by oandrew** [comment]
> I'm not entirely sure if we should include the actual device labels (this can be done conditionally), split the file into 4, or keep it as is.

Updated to use conditional label based on `productId`

p.s. 
IMO ideally zwave-js would support  per-device `label` / `description` properties to simplify this. e.g.
```
"devices": [
		//  (2nd Generation Assure Keyed Push Button Biometric Deadbolt)
		{
			"productType": "0x8107",
			"productId": "0x49d1",
			"zwaveAllianceId": 4936,
                        "label": "YRD410-F-ZW3",
                        "description": "Assure 2 Keyed Push Button Biometric Deadbolt"
		},
```

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**